packages/api-client/src/v2/hooks/use-document-watcher.test.ts

```diff
@@ -1,9 +1,8 @@
-import assert from 'node:assert'
 import { setTimeout } from 'node:timers/promises'
 
 import { createWorkspaceStore } from '@scalar/workspace-store/client'
 import { type FastifyInstance, fastify } from 'fastify'
-import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+import { assert, beforeEach, describe, expect, it, vi } from 'vitest'
```

Comment:
This is unrelated, however,  I have this test file failing locally with the same error reported here https://github.com/scalar/scalar/pull/7099#discussion_r2432306772,
so I switched `assert` from `node:assert` to `vitest` like it did there